cmd/agent/main.go

```diff
@@ -70,6 +73,7 @@ func (o *GrpcProxyAgentOptions) Flags() *pflag.FlagSet {
 	flags.StringVar(&o.caCert, "ca-cert", o.caCert, "If non-empty the CAs we use to validate clients.")
 	flags.StringVar(&o.proxyServerHost, "proxy-server-host", o.proxyServerHost, "The hostname to use to connect to the proxy-server.")
 	flags.IntVar(&o.proxyServerPort, "proxy-server-port", o.proxyServerPort, "The port the proxy server is listening on.")
+	flags.StringVar(&o.serviceAccountToken, "service-account-token", o.serviceAccountToken, "If non-empty authorize each call to the proxy-server with token file's content.")
```

Comment:
nit: how about "If non-empty proxy agent uses this token to prove its identity to the proxy server"?

Reply:
string has been replaced as suggested.